checker/votour.ml

```diff
@@ -10,6 +10,7 @@ open Values
 
 (** {6 Interactive visit of a vo} *)
 
+[@@@ocaml.warning "-52"]
```

Comment:
Instead of disabling this warning please fix it, eg by replacing the try with block with
``` ocaml
    match int_of_string l with
    | v ->
      if v < 0 || v >= max then
        let () =
          Printf.printf "Out-of-range input! (only %d children)\n%!" max in
        read_num max
      else Some v
    | exception Failure _ ->
      Printf.printf "Unrecognized input! <n> enters the <n>-th child, u goes up 1 level, x exits\n%!";
      read_num max
```